der-moderne-staat.csl

```diff
@@ -0,0 +1,251 @@
+        </if>
+        <else-if type="book graphic motion_picture song manuscript thesis" match="any">
+          <text macro="legal_case"/>
+          <group prefix=" " delimiter=" ">
+            <text macro="title" suffix=","/>
+            <text macro="edition"/>
+            <text macro="editor" suffix=","/>
+          </group>
+          <text prefix=" " suffix="." macro="publisher"/>
+        </else-if>
+        <else-if type="chapter" match="any">
+          <text macro="title" prefix=" " suffix=","/>
+          <group prefix=" " delimiter=" ">
+            <text term="in" suffix=":"/>
+            <text macro="editor" suffix=","/>
+            <text variable="container-title" font-style="normal" suffix=","/>
+            <text variable="collection-title" suffix=","/>
+            <group suffix=".">
+              <text macro="publisher" suffix=", "/>
+              <label variable="page" form="short" suffix=" "/>
+              <text variable="page"/>
+            </group>
+          </group>
+        </else-if>
+        <else-if type="paper-conference" match="any">
+          <text macro="title" prefix=" " suffix=","/>
+          <group prefix=" " delimiter=", ">
+            <text variable="event" prefix="paper presented at " font-style="italic"/>
+            <text variable="publisher-place"/>
+            <date variable="issued" suffix=".">
+              <date-part name="month" form="long" suffix=" "/>
+              <date-part name="day" suffix=", "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </else-if>
+        <else-if type="article-newspaper article-magazine" match="any">
+          <text macro="title" prefix=" " suffix=", "/>
+          <text variable="container-title" font-style="italic" prefix=" " suffix=", "/>
+          <text variable="collection-title" suffix=", "/>
+          <group suffix="." delimiter=", ">
+            <text macro="publisher"/>
+            <date variable="issued">
+              <date-part name="month" form="long" suffix=" "/>
+              <date-part name="day" suffix=", "/>
+              <date-part name="year"/>
+            </date>
+            <group>
+              <label variable="page" form="short" suffix=" "/>
+              <text variable="page"/>
+            </group>
+          </group>
+        </else-if>
+        <else>
+          <group prefix=" " delimiter=" " suffix=",">
+            <text macro="title"/>
+            <text macro="editor"/>
+          </group>
+          <group delimiter=", " prefix=" ">
+            <text variable="container-title" font-style="normal"/>
+            <group delimiter=", ">
+              <text variable="volume"/>
+              <text variable="issue" form="short"/>
+            </group>
+            <text variable="page" form="short" prefix="S. "/>
+          </group>
+        </else>
+      </choose>
+      <text prefix=" " macro="access" suffix="."/>
```

Comment:
The `suffix="."` seems to be too much, because the `layout` has already a suffix.